test/referencetests.jl

```diff
@@ -10,42 +9,56 @@ a2 = a2[1:20]
 a3 = a3[1:20]
 mus = mus[1:20]
 
+#Somehow fig seems to get converted in AbstractArray{<:Colorant} of different sizes, depending on 
+#what display is used; the function below is a workaround through file saving/loading. Not 
+#elegant, but it works.
+function save_load_remove(fig)
```

Comment:
I asked on Slack and one can use `colorbuffer(fig)` to directly get a `Matrix` of Colorant.

Reply:
But anyway that's off topic from this MR. Let's just get this merged and we can apply an eventual better solution in a separate MR.

Reply:
Solution seems to be `colorbuffer(fig; px_per_unit=2)` ?

Reply:
Added a new commit; needed some more tweaking, but I think this version is cleaner than before.